requirements.txt

```diff
@@ -0,0 +1,14 @@
+click==7.1.2
+dnspython==2.1.0
+email-validator==1.1.2
+Flask==1.1.2
+flask-mongoengine==1.0.0
+Flask-WTF==0.14.3
+idna==3.1
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+mongoengine==0.23.0
+pymongo==3.11.3
+Werkzeug==1.0.1
+WTForms==2.3.3
```

Comment:
requirements.txt for web-server skal ligge inni `web-server` mappen. Det ferdige prosjektet vil kun trenge å starte docker bildene, uten å måtte installere requirements filene.

Egentlig kunne hver mappe vært et eget github repository, men for å gjøre ting enkelt så har vi alt i samme.